config-files/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+### PR Checklist
+
+- [ ] JIRA Ticket(s) included in PR Title and are formatted correctly if applicable (JIRA-123)
```

Comment:
This can be rephrased: `JIRA Ticket(s) are included in the PR Title and are formatted correctly: JIRA-123`

What if there are no associated JIRA tickets?